src/index.ts

```diff
@@ -15,9 +15,7 @@ import { readFile } from '@/utils/fs';
     console.log(gimbal);
   }
 
-  // cannot import package.json as it is not within the rootDir (aka, src/)
-  const rawPackageJson = await readFile(path.join(__dirname, '../package.json'), 'utf8');
-  const packageJson = JSON.parse(rawPackageJson);
+  const packageJson = await readPkg();
```

Comment:
Does it need to go up one level? I'm seeing in that package it does only:

https://github.com/sindresorhus/read-pkg/blob/7a86b115b300fb4f4fae392376bb2a389910bc45/index.js#L16

Reply:
Not needed. By default, it tries to find the project's root package.json.
Right now, it is fetching the correct package.json.
We for any other package.json in a nested folder, we could provide `cwd` property when needed.

Reply:
You're right, `process.cwd()` here !== `__dirname`.